contracts/mocks/ERC20Mock.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.9;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract ERC20Mock is ERC20, Ownable {
```

Comment:
We already have an ERC20 mock. Let's avoid adding this duplicated mock please
https://github.com/lens-protocol/modules/blob/master/contracts/mocks/ACurrency.sol